core/src/main/java/nl/nn/adapterframework/pipes/XmlValidator.java

```diff
@@ -265,12 +271,17 @@ protected PipeForward determineForward(String resultEvent, IPipeLineSession sess
 	}
 
 	@Deprecated
-	private String getMessageToValidate(Object input, IPipeLineSession session) throws PipeRunException {
-		String inputStr = input.toString();
-		if (XmlUtils.isWellFormed(inputStr, "Envelope")) {
+	private String getMessageToValidate(Message message, IPipeLineSession session) throws PipeRunException {
+		String input;
+		try {
+			input = message.asString();
+		} catch (IOException e) {
+			throw new PipeRunException(this, getLogPrefix(session)+"cannot open stream", e);
+		}
+		if (XmlUtils.isWellFormed(input, "Envelope")) {
 			String inputRootNs;
 			try {
-				inputRootNs = transformerPoolGetRootNamespace.transform(inputStr, null);
+				inputRootNs = transformerPoolGetRootNamespace.transform(input, null);
```

Comment:
is het niet handiger als dit een Message is?

Reply:
het is handiger als die hele method niet gebruikt wordt, dus als de deprecated attribute soapNamespace niet gebruikt wordt.